my-app/src/components/cart-card.js

```diff
@@ -20,7 +31,7 @@ const CartCard = ({qty,price,author,id,title}) => {
                         <button style = {{width:'1rem'}} onClick = {() => updateQuantity(id,state.encodedToken,'increment',dispatch,qty)}>+</button>
                         <button style = {{width:'1rem'}} disabled = {qty === 1} onClick = {() => updateQuantity(id,state.encodedToken,'decrement',dispatch,qty)}>-</button>
                         <button className = "btn btn-error m2-top full-width" onClick = {() => deleteProduct(id,state.encodedToken,dispatch)}>Remove from Cart</button>
```

Comment:
Good job on disabling the decrement button after the count goes below 1. I will add this to my ecom app  as well.